org.elysium.parent/org.elysium.ui/src/org/elysium/ui/hyperlinks/LilyPondLocationIfFileProvider.java

```diff
@@ -0,0 +1,36 @@
+package org.elysium.ui.hyperlinks;
+
+import java.util.List;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.xtext.nodemodel.INode;
+import org.eclipse.xtext.nodemodel.util.NodeModelUtils;
+import org.eclipse.xtext.resource.DefaultLocationInFileProvider;
+import org.eclipse.xtext.util.ITextRegion;
+import org.eclipse.xtext.util.TextRegion;
+import org.elysium.lilypond.Include;
+import org.elysium.lilypond.LilypondPackage;
+
+public class LilyPondLocationIfFileProvider extends DefaultLocationInFileProvider {
```

Comment:
Please rename to `LilyPondLocationInFileProvider`.

Reply:
fixed